### PR TITLE
Update give Cloud access to a GCP user

### PR DESCRIPTION
**Pull request by gilbennun**

### CODE REVIEW

**Comment by bmansheim** [comment]
@Aviad-Abutbul I'm happy to review the style and formatting. Can you confirm that we need to make this update?

**Comment by kaitlynmichael** [comment]
@gilbennun I need you to sign the contributor agreement before I can merge this change.

File: content/rc/administration/account-team-settings.md [hunk 1]
```diff
@@ -36,13 +36,22 @@ Team members can have different roles to the account:
...
+If you subscribed to Redis Enterprise Cloud using GCP Marketplace, you can manage your team from the **IAM section** of the GCP console.
+To grant Redis Cloud access to a GCP user, select the **Add** button to add a member, insert the email address, and assign one of these roles to the user:
+
+ serviceusage.serviceUsageViewer & redisenterprisecloud.viewer - these roles should be assigned to member designated as viewer
+ serviceusage.serviceUsageViewer & redisenterprisecloud.admin -  these roles should be assigned to member designated as owner
+ 
+In case these roles do not appear, add a role to your project by doing the following:
+1. Select **Manage Roles** 
+1. Search the role on the **filter table** field ( for example: "service usage viewer" or "redis enterprise cloud admin")
+1. Mark the checkbox
```
Comment (lanceleonard) [inline content/rc/administration/account-team-settings.md:49]:
```suggestion
1. Place a check in the checkbox
```

File: content/rc/administration/account-team-settings.md [hunk 2]
```diff
@@ -36,13 +36,22 @@ Team members can have different roles to the account:
 
 ### Team management for GCP Marketplace customers
 
-If you subscribed to Redis Enterprise Cloud using GCP Marketplace, you can manage your team from the IAM section of the GCP console.
-To grant Redis Cloud access to a GCP user, assign one of these roles to the user:
 
-- **Viewer** - serviceusage.serviceUsageViewer and redisenterprisecloud.viewer
-- **Owner** - serviceusage.serviceUsageViewer and redisenterprisecloud.admin
+If you subscribed to Redis Enterprise Cloud using GCP Marketplace, you can manage your team from the **IAM section** of the GCP console.
+To grant Redis Cloud access to a GCP user, select the **Add** button to add a member, insert the email address, and assign one of these roles to the user:
```
Comment (lanceleonard) [inline content/rc/administration/account-team-settings.md:41]:
```suggestion
To grant Redis Cloud access to a GCP user, select the **Add** button to add a member, insert the email address, and then assign one of the following roles to the user:
```

File: content/rc/administration/account-team-settings.md [hunk 3]
```diff
@@ -36,13 +36,22 @@ Team members can have different roles to the account:
...
+ serviceusage.serviceUsageViewer & redisenterprisecloud.admin -  these roles should be assigned to member designated as owner
+ 
+In case these roles do not appear, add a role to your project by doing the following:
+1. Select **Manage Roles** 
+1. Search the role on the **filter table** field ( for example: "service usage viewer" or "redis enterprise cloud admin")
+1. Mark the checkbox
+1. Select **Create role from selection** and select **Create**
+1. Go back to **IAM** to add a member and assign the desried roles.
 
 Users must log in using SSO to Redis Cloud at least once for them to be added to the team.
```
Comment (lanceleonard) [inline content/rc/administration/account-team-settings.md:53]:
```suggestion
Users are not added to the team until they sign in to Redis Cloud using their single-sign on (SSO) credentials.
```

File: content/rc/administration/account-team-settings.md [hunk 4]
```diff
@@ -36,13 +36,22 @@ Team members can have different roles to the account:
...
+This cam be done by clicking on the button **manage on provider**
```
Comment (lanceleonard) [inline content/rc/administration/account-team-settings.md:54]:
```suggestion
Use the **manage on provider** button to do so. 
```

**Review by lanceleonard — APPROVED** [review]
LGTM, with suggested edits.

**Pull request merged**